docs-v3/elements/source/includes/_components.md

```diff
@@ -346,6 +346,10 @@ Displays the price, discount price and unit price ( eg. 2$/100ml, if applicable
 
 Displays the full-size version of the active product image. With some configuration options it can be also used to change the active product image.
 
+<aside class="notice">
+<code>ProductImage</code> component must be placed in a container with fixed width and height.
```

Comment:
Zmień proszę na:
```
<code>ProductImage</code> component must be placed in a container with fixed width and height and it will grow to fill that container
```
(tutaj i w pozostałych miejscach)